admission_control/admission-control-service/src/admission_control_service.rs

```diff
@@ -61,7 +58,8 @@ impl AdmissionControlService {
             ledger_consistency_proof,
         ) = self
             .storage_read_client
-            .update_to_latest_ledger(rust_req.client_known_version, rust_req.requested_items)?;
+            .update_to_latest_ledger_async(rust_req.client_known_version, rust_req.requested_items)
```

Comment:
Every other _async got removed, but here you added one?

Reply:
Ah! no actually that comes from the storage Grpc interface. Since that interface hasn't been converted yet I preemptively changed from using the blocking to the async version of that method.